### ISSUE TITLE
dataio-shovel "write frame+dependencies to file" outputs only the dependencies, not the current frame (Trac #2125)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#2117]
In the help of dataio-shovel, it says:

```text
w                Write entire frame (binary) to file
W                Write entire frame (binary) to file, along with all frames
                   on which it depends
```
`w` works as intendent, but `W` doesn't.
Selecting e.g. a P-frame and pressing `W` writes the GCDQ frames on which the P frame depends, but not the P frame itself.

Migrated from https://code.icecube.wisc.edu/ticket/2125
```json
{
    "status": "closed", 
    "changetime": "2019-01-11T22:01:58", 
    "description": "In the help of dataio-shovel, it says:\n{{{\nw                Write entire frame (binary) to file\nW                Write entire frame (binary) to file, along with all frames\n                   on which it depends\n}}}\n`w` works as intendent, but `W` doesn't.\nSelecting e.g. a P-frame and pressing `W` writes the GCDQ frames on which the P frame depends, but not the P frame itself.", 
    "reporter": "thomas.kintscher", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1547244118311846", 
    "component": "combo core", 
    "summary": "dataio-shovel \"write frame+dependencies to file\" outputs only the dependencies, not the current frame", 
    "priority": "normal", 
    "keywords": "", 
    "time": "2017-12-19T12:48:06", 
    "milestone": "", 
    "owner": "cweaver", 
    "type": "defect"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20171219T15:07:16: 
* cweaver changed status from "new" to "accepted"
* cweaver changed owner from "" to "cweaver"

**Comment by icecube-trac** [comment]
Trac update at 20171219T16:40:47: 
* cweaver changed resolution from "" to "fixed"
* cweaver commented:

> In changeset 3028:
> 
> ```text
> #!CommitTicketReference repository="IceTray" revision="3028"
> Fix an apparent copy-paste bug.
> Fixes #2117
> ```


* cweaver changed status from "accepted" to "closed"

**Comment by icecube-trac** [comment]
Trac update at 20190111T22:01:58: nwhitehorn commented:

> In changeset 169261:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="169261"
> Fix an apparent copy-paste bug.
> Fixes #2117
> ```
